client/my-sites/email/inbox/mailbox-selection-list/index.jsx

```diff
@@ -51,6 +55,31 @@ const MailboxItemIcon = ( { mailbox } ) => {
 	return null;
 };
 
+const getProvider = ( { mailbox } ) => {
+	if ( isTitanMailAccount( mailbox ) ) {
+		return 'titan';
+	}
+
+	if ( isGoogleEmailAccount( mailbox ) ) {
+		return 'google';
+	}
+
+	if ( isEmailForwardAccount( mailbox ) ) {
+		return 'forward';
+	}
+
+	return null;
+};
+
+const trackEvent = ( { mailbox, app, context } ) => {
+	const provider = getProvider( mailbox );
+	recordEmailActionTrackEvent( {
+		app,
+		context,
+		provider,
+	} );
```

Comment:
I haven't debugged this, but this is recording `null` for the `provider` for Google and Professional Email mailboxes.

Reply:
Ups. Solved, it was a last-minute change.